doc/swss-schema.md

```diff
@@ -159,8 +159,29 @@ and reflects the LAG ports into the redis under: `LAG_TABLE:<team0>:port`
     ;Status: Mandatory
     key           = ROUTE_TABLE:prefix
     nexthop       = *prefix, ;IP addresses separated “,” (empty indicates no gateway)
-    intf          = ifindex? PORT_TABLE.key  ; zero or more separated by “,” (zero indicates no interface)
-    blackhole     = BIT ; Set to 1 if this route is a blackhole (or null0)
+    ifname        = ifindex? PORT_TABLE.key  ; zero or more separated by “,” (zero indicates no interface)
+    weight        = weight_list              ; List of weights.
+    nexthop_group = string ; index within the NEXT_HOP_GROUP_TABLE, used instead of nexthop and intf fields
+
+---------------------------------------------
+
+###### LABEL_ROUTE_TABLE
+; Defines schema for MPLS label route table attributes
+key           = LABEL_ROUTE_TABLE:mpls_label ; MPLS label
+; field       = value
+nexthop       = STRING                   ; Comma-separated list of nexthops.
+ifname        = STRING                   ; Comma-separated list of interfaces.
+weight        = STRING                   ; Comma-separated list of weights.
+nexthop_group = string ; index within the NEXT_HOP_GROUP_TABLE, used instead of nexthop and intf fields
+
+---------------------------------------------
+### NEXT_HOP_GROUP_TABLE
```

Comment:
NEXT_HOP_GROUP_TABLE doesn't has VNI_LABEL and router_mac information for overlay routes(for instance EVPN Type-5 prefix routes).
Without these fields we will not be able to support overlay route nexthop groups.

Reply:
I've attempted to add this, but hit an issue where the call to createRemoteVtep in routeorch for each next hop requires the VRF of the route as a parameter. Currently the next hop group table has no VRF information, and each group could be used for multiple VRFs. Since it's not clear what the best solution would be (either to add VRF info to the next hop group table for this case, or to wait until the route is also created to do the extra programming), and this wouldn't be as well tested since we wouldn't be using this feature, I suggest putting those attributes out of scope at this time.

However, if you are planning to use this new feature soon and would need the VNI and router_mac attributes, do let me know, and we can work out what the preferred way of implementing this would be.